util/topgen/intermodule.py

```diff
@@ -108,12 +143,9 @@ def elab_intermodule(topcfg: OrderedDict):
                                                  rsp_module, rsp_signal)
 
             # determine the signal name
-            sig_name = "im{uid}_{req_s}".format(uid=uid,
-                                                req_s=req_struct['struct'])
-            sig_name = intersignal_format(uid, req_struct, rsp_struct)
 
-            req_struct["top_signame"] = sig_name
             rsp_struct["top_signame"] = sig_name
+            rsp_struct["index"] = -1 if req_struct["width"] == 1 else i
 
             # Add to definitions
             if "package" in req_struct:
```

Comment:
does this still need to be here..?

Reply:
I thought I removed this... :) Thanks for catching it